cross/libicu/Makefile

```diff
@@ -1,11 +1,9 @@
 PKG_NAME = libicu
-PKG_VERS_MAJOR = 67
-PKG_VERS_MINOR = 1
-PKG_VERS = $(PKG_VERS_MAJOR).$(PKG_VERS_MINOR)
+PKG_VERS = 68.2
```

Comment:
Please keep PKG_VERS_MAJOR and PKG_VERS_MINOR that are useful to automatically search for new minor versions. Like for instance https://github.com/GuillaumeSmaha/spksrc-check-update

Reply:
I was not aware about a working update checker. 
IMHO such an update checker would work best when PKG_VERS is defined as plain full version (without any variables).
It makes package updates simple and clear with PKG_VERS defined in one single line of code.

Reply:
@hgy59 You are correct, and I check code, it uses `PKG_VERS` only. So go for this change, no issue. I have requested @Diaoul for a new repository in organisation so that this tool becomes visible and more used/maintained!